docs/adr-0002-publishing_statistics_with_analysis_basis.md

```diff
@@ -63,3 +63,24 @@ This has an impact on the structure of the datasets Jetstream publishes and adds
 Data scientists might access result tables via Redash (and possibly Looker). Information about the analysis basis used is pretty important.
 
 **Option 4** wins for having analysis basis information as part of the results and for not being a breaking change for Experimenter.
+
+
+## Changes when working with statistics tables
+
+### Option 1: Support computing metrics for multiple analysis bases
+
+By adding support for different analysis basis, it is also possible to configure metrics to be computed for multiple analysis bases. For example, the days_of_use metric can be computed based on exposures and enrollments. This means that statistics tables have potentially multiple results of the same statistic for a single metric, depending on the whether it has been computed for multiple analysis bases. This introduces a breaking change to the previous contract of the statistic tables, so that grouping by the analysis basis would be required as well.
+
+:- Change of the contract with the stats tables. Relevant for Experimenter console and data scientists that work with the stats tables directly
```

Comment:
We don't directly access the stats tables from experimenter, but the output into the gcs bucket. Would these have an additional field that says the `analysis_basis` and there could be more than one of these in this case?

Reply:
Yes, the GCS output will also have an additional `analysis_basis` column. The GCS output and views have the same structure as the underlying statistics tables but just combine data of multiple days/weeks/...

Reply:
Ok, this would be a breaking change for experimenter, but it wouldn't be too hard to fix. Would like to see what @tdsmith thinks. Might be worth breaking the "contract" to be able to do both analysis basis per metric

Reply:
I'm in favour; I think this makes a bunch of things more sensible.